tests/add.bats

```diff
@@ -321,3 +321,16 @@ stuff/mystuff"
   run_buildah 125 add --retry-delay=0.142857s --retry=14 --cert-dir ${TEST_SCRATCH_DIR} $cid https://localhost:${HTTP_SERVER_PORT}/randomfile
   assert "$output" =~ "retrying in 142.*ms .*14/14.*"
 }
+
+@test "add file with IMA xattr" {
+    if getfattr -d -n 'security.ima' /usr/libexec/catatonit/catatonit | grep -q ima; then
```

Comment:
```suggestion
    if ! getfattr -d -n 'security.ima' /usr/libexec/catatonit/catatonit | grep -q ima; then
```

Reply:
Also note that this will not be tested in podman CI, because that only runs `bud.bats`. When this is vendored into podman, a new test will be required.